mule-migration-tool-e2e-tests/src/test/resources/e2e/http/http3Mvn/output_nc/src/main/mule/mule-config.xml

```diff
@@ -0,0 +1,72 @@
+            <http:error-response>
+                <http:headers><![CDATA[#[vars filterObject ($$ startsWith 'outbound_') mapObject {($$ dw::core::Strings::substringAfter '_'): $}]]]></http:headers>
+            </http:error-response>
+        </http:listener>
+
+
+        <!--Migration WARN: When migrating copy properties headers sent by the user or implicit properties are not converted to variables, if there is any specific reference to them as outbound properties they will be need to be migrated manually-->
+        <!--<copy-properties xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:migration="migration" propertyName="*" />-->
+        <set-variable variableName="vars.outbound_LOCAL_CERTIFICATES" value="#[([message.attributes.clientCertificate])]" />
+        <set-variable variableName="vars.outbound_PEER_CERTIFICATES" value="#[([message.attributes.clientCertificate])]" />
+        <set-variable variableName="vars.outbound_http.client.cert" value="#[message.attributes.clientCertificate]" />
+        <set-variable variableName="vars.outbound_http.context.path" value="#[(if (endsWith(message.attributes.listenerPath, '/*')) message.attributes.listenerPath[0 to -3] default '/' else message.attributes.listenerPath)]" />
+        <set-variable variableName="vars.outbound_http.headers" value="#[message.attributes.headers]" />
+        <set-variable variableName="vars.outbound_http.listener.path" value="#[message.attributes.listenerPath]" />
+        <set-variable variableName="vars.outbound_http.method" value="#[message.attributes.method]" />
+        <set-variable variableName="vars.outbound_http.query.params" value="#[message.attributes.queryParams]" />
+        <set-variable variableName="vars.outbound_http.query.string" value="#[message.attributes.queryString]" />
+        <set-variable variableName="vars.outbound_http.relative.path" value="#[(message.attributes.requestPath[1 + sizeOf(if (endsWith(message.attributes.listenerPath, '/*')) message.attributes.listenerPath[0 to -3] default '/' else message.attributes.listenerPath) to -1])]" />
+        <set-variable variableName="vars.outbound_http.remote.address" value="#[message.attributes.remoteAddress]" />
+        <set-variable variableName="vars.outbound_http.request" value="#[message.attributes.requestPath]" />
+        <set-variable variableName="vars.outbound_http.request.path" value="#[message.attributes.requestPath]" />
+        <set-variable variableName="vars.outbound_http.request.uri" value="#[message.attributes.requestUri]" />
+        <set-variable variableName="vars.outbound_http.scheme" value="#[message.attributes.scheme]" />
+        <set-variable variableName="vars.outbound_http.uri.params" value="#[message.attributes.uriParams]" />
+        <set-variable variableName="vars.outbound_http.version" value="#[message.attributes.version]" />
+    
+        <remove-variable variableName="outbound_http.*" />
+
+        <compatibility:multipart-to-vars partName="*">

```

Comment:
shouldn't this be part of having the step at the end remove anything that is left? I think we can also have a task to properly handle this element (even put an error saying that we are not able to migrate it properly) but I think this can happen with more elements (just that they did not appear in these tests) so we should remove anything that is left and then have tasks for each element to handle properly

Reply:
Created task https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000yxB6fYAE/view